pkg/secrethub/credentials/credentials.go

```diff
@@ -11,6 +11,8 @@ type Verifier interface {
 	Type() api.CredentialType
 	// AddProof adds the proof of this credential's possession to a CreateCredentialRequest.
 	AddProof(req *api.CreateCredentialRequest) error
+	// Fingerprint returns the key identifier by which the server can identify the verifier.
+	Fingerprint() (string, error)
```

Comment:
Question: what made you decide to expand the interface while it looks like getting the fingerprint can already be achieved with the current interface (using `Verifier` and `Type` functions)?

From your code above:

```go
verifier, err := c.Verifier()
if err != nil {
	return "", err
}

 return api.GetFingerprint(c.Type(), verifier)
```

Maybe I'm missing something.